client/src/components/Modal/MapModal.js

```diff
@@ -3,9 +3,14 @@ import PropTypes from 'prop-types';
 import { View, Modal, TouchableOpacity, Dimensions, TouchableWithoutFeedback, Text } from 'react-native';
 
 import MapView, { Marker } from 'react-native-maps';
-import { IconMarker } from '../../components/MapMarker/';
+import MapViewDirections from 'react-native-maps-directions';
+import { IconMarker, MyLocationMarker } from '../../components/MapMarker/';
+
 import MapDelta from '../../selectors/MapDelta';
 
+import { GOOGLEMAPS_API_KEY } from '../../config/apis';
```

Comment:
why is this hard-coded in `Events/SetResponse.js`?

Reply:
how did I miss that